src/hourly-weather.ts

```diff
@@ -393,6 +393,7 @@ export class HourlyWeatherCard extends LitElement {
             .precipitation=${precipitation}
             .icons=${!!config.icons}
             .colors=${colorSettings.validColors}
+            .borders=${config.borders}
```

Comment:
Like the `colors` config, we must validate the input here before using it. Otherwise, this opens a security hole for folks to inject arbitrary persisted script in the card CSS. You should be able to reuse the color validation logic that we already have.